languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -21941,6 +21941,24 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rule>
         <rulegroup id="DIENSTAG_ABEND" name="'Dienstag abend (Dienstagabend)">
           <rule>
+            <antipattern>
+                    <token regexp="yes">&wochentage;</token>
+                    <token min="0" regexp="yes">und|oder</token>
+                    <token regexp="yes">&wochentage;</token>
+                    <token min="0" regexp="yes">und|oder</token>
+                    <token postag="PRP:.+|ART:.+|ADJ:.+" postag_regexp="yes"/>
+                    <token min="0" regexp="yes">&wochentage;</token>
+                    <token regexp="yes">&tageszeitangabenklein;</token>
+            </antipattern>
+            <antipattern>
```

Comment:
als AP können das erste und das zweite zusammengefasst werden...im AP hat man ja keine suggestion und die war ja das eigentliche problem. dann hat man nicht so viele APs in der regel